.github/workflows/ci.yml

```diff
@@ -116,7 +116,40 @@ jobs:
       - name: Prereqs
         run: python ./prereqs.py --install
       - name: Build and Test
```

Comment:
Maybe we should remove the `and test` from the name of this task?

Reply:
Oh, I guess it still runs the tests (I always get mixed up with --no-check and --no-test).